Fixed an issue with resolving packages with names defined by function calls in ``setup.py``.
